### ISSUE TITLE
hdmi-csi no data

### ISSUE THREAD

**Issue by zhjygit** [ehay64/hdmi-capture#1]
I made a test, the target PC transmits data via hdmi, raspberry receives data via hdmi-cis connected to hdmi port of target PC;
while, i recevie data at remote PC via putty to raspberry,everythin is OK, phtots and videos run well.
However, the target PC transmits data via vga->hdmi(some specia devices), raspberry receives data via hdmi-cis connected to hdmi port of target PC, the results is that no data from raspberry, and raspberry is no response, finally, i shutdown the raspberry.

In fact, what i do above is something about  MITM test, for example , raspberry receives data via hdmi-cis connected to hdmi port of a projector, one device named screen scrb can finish.

In summary, CSI-HDMI of raspberry cannot reveive data from VGA->HDMI.

**Comment by ehay64** [comment]
In summary, your setup is as follows (correct me if I am mistaken):
Host PC -> VGA to HDMI adapter -> HDMI-CSI bridge -> Raspberry Pi

Are you able to verify the resolution and timing settings that the VGA to HDMI adapter is set to? The HDMI-CSI bridge supports a very limited number of resolutions, it's possible the VGA to HDMI adapter is outputting a resolution beyond what the HDMI-CSI bridge can handle (at least with only 2 lanes).

See if you can force the VGA to HDMI adapter to a lower resolution (maybe 1280x720) at a lower refresh rate (maybe 30Hz). `v4l2-ctl --query-dv-timings` should show if the HDMI-CSI bridge can read the timings properly.

**Comment by zhjygit** [comment]
Thank you,  I have solved this problem.
